### PR TITLE
Remove support for AiiDA 0.x in start.ipynb

### PR DESCRIPTION
**Pull request by danielhollas**
Removed code that was meant to support AiiDA <1.0 :scream: 
+ other minor cleanups

### CODE REVIEW

File: start.ipynb [hunk 1]
```diff
@@ -33,72 +37,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import aiida\n",
-    "\n",
-    "home = None\n",
-    "\n",
-    "if aiida.__version__ < \"1.0.0\":\n",
```
Comment (danielhollas) [inline start.ipynb:40]: Pretty sure we don't support AiiDA this old :smile:

File: start.ipynb [hunk 2]
```diff
@@ -23,8 +23,13 @@
    "source": [
     "import urllib.parse as urlparse\n",
     "\n",
-    "url = urlparse.urlsplit(jupyter_notebook_url)  # noqa: F821\n",
-    "parsed_url = urlparse.parse_qs(url.query)"
+    "# variable `jupyter_notebook_url` is injected by appmode\n",
+    "try:\n",
+    "    url = urlparse.urlsplit(jupyter_notebook_url)\n",
+    "except NameError:\n",
+    "    parsed_url = \"\"\n",
```
Comment (danielhollas) [inline start.ipynb:30]: This is just to make the notebook runnable outside of appmode

**Comment by danielhollas** [comment]
@unkcpz it looks like you've done the same thing in the [chore/xx/remove/aiida-0x-support branch](https://github.com/aiidalab/aiidalab-home/tree/chore/xx/remove-aiida-0x-support) but did not open a PR.

**Review by yakutovicha — APPROVED** [review]
Thanks @danielhollas, very happy to approve this PR!

File: terminal.ipynb [hunk 3]
```diff
@@ -33,23 +33,23 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 2",
```
[the remark this answers is not in the document]
Comment (danielhollas) [reply terminal.ipynb:36]: Haha, OMG indeed. Thanks for approving!

**Pull request merged**